rolling/distribution.yaml

```diff
@@ -2817,6 +2817,21 @@ repositories:
       url: https://github.com/DFKI-NI/mir_robot.git
       version: rolling
     status: developed
+  mobile_catkin_modules_build_development_tools:
+    doc:
+      type: git
+      url: https://github.com/Application-UI-UX/mobile_catkin_modules_build_development_tools.git
```

Comment:
I'm pretty confused about this repository.  It has "catkin_modules" in the name, but it is being released for ROS 2 (which doesn't support catkin).  It also has a bunch of generated code checked into it, so it doesn't look like a normal ROS package.  Can you please explain more about what the code in this repository does?

Reply:
Yes, there are generated code. The description is in the name, it is for ros and ros2. It is a mobile development tool.  The general short purpose is to be able to have connections to robots from mobile. The catkin is not for the robot itself it is for the mobile.